website/components/Header.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Button from '@leafygreen-ui/button';
+import DownloadIcon from '@leafygreen-ui/icon/dist/Download';
+import { Tabs, Tab } from '@leafygreen-ui/tabs';
+import { css } from 'emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { H2 } from '@leafygreen-ui/typography';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+import { useViewportSize } from '@leafygreen-ui/hooks';
+import { BaseLayoutProps } from 'utils/types';
+import CodeDocs from 'components/CodeDocs';
+import ReactIcon from 'components/svgs/ReactIcon';
+import SketchIcon from 'components/svgs/SketchIcon';
+
+const componentsStyle = css`
+  height: 16px;
+  color: ${uiColors.gray.dark1};
+  font-size: 12px;
+  letter-spacing: 0.4px;
+  line-height: 16px;
+  text-transform: uppercase;
+  font-weight: bold;
+}
+`;
+
+const marginBotton = css`
+  margin-bottom: 36px;
+`;
+
+const flexContainer = css`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const caps = css`
+  text-transform: capitalize;
+`;
+
+const tabNameContainer = css`
+  display: flex;
+  align-items: center;
+`;
+
+const iconStyle = css`
+  margin-right: ${spacing[1]}px;
+  flex-shrink: 0;
+`;
+
+function Header({ component, changelog, readme }: BaseLayoutProps) {
+  const viewport = useViewportSize();
+  const isMobile = viewport?.width < breakpoints.Tablet;
```

Comment:
sketchy null comparison
```suggestion
  const isMobile = viewport === null ? viewport.width < breakpoints.Tablet : false;
```

Reply:
oops, yep

Reply:
👍